internal/sdkv2provider/resource_cloudflare_access_application.go

```diff
@@ -213,6 +215,7 @@ func resourceCloudflareAccessApplicationUpdate(ctx context.Context, d *schema.Re
 		SkipInterstitial:         cloudflare.BoolPtr(d.Get("skip_interstitial").(bool)),
 		AppLauncherVisible:       cloudflare.BoolPtr(d.Get("app_launcher_visible").(bool)),
 		ServiceAuth401Redirect:   cloudflare.BoolPtr(d.Get("service_auth_401_redirect").(bool)),
+		AllowAuthenticateViaWarp: cloudflare.BoolPtr(d.Get("allow_authenticate_via_warp").(bool)),
```

Comment:
this looks like it is always sending a value which is causing the API throw a 400. do we always want to send it? should the backing service support a `false` value even when it isn't toggled on the org?

```
POST /client/v4/zones/0da42c8d2132a9ddaf714f9e7c920711/access/apps HTTP/1.1
Host: api.cloudflare.com
User-Agent: terraform-provider-cloudflare/dev terraform-plugin-sdk terraform/1.5.4
Content-Length: 566
Content-Type: application/json
X-Auth-Email: [redacted]
X-Auth-Key: [redacted]
Accept-Encoding: gzip

{"app_launcher_visible":true,"auto_redirect_to_identity":false,"domain":"qghbheofcs.terraform.cfapi.net","enable_binding_cookie":false,"http_only_cookie_attribute":false,"name":"qghbheofcs","private_address":"","self_hosted_domains":null,"service_auth_401_redirect":false,"session_duration":"24h","skip_interstitial":false,"type":"self_hosted","allow_authenticate_via_warp":false,"landing_page_design":{"title":"","message":"","image_url":"","button_color":"","button_text_color":""},"app_launcher_logo_url":"","header_bg_color":"","bg_color":"","footer_links":null}

HTTP/1.1 400 Bad Request
Date: Thu, 15 Feb 2024 04:15:05 GMT
Content-Type: application/json; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive
CF-Ray: 855aba48cf57550f-SYD
CF-Cache-Status: DYNAMIC
Content-Encoding: gzip
Set-Cookie: __cflb=0H28vgHxwvgAQtjUGUFqYFDiSDreGJnV3YzQozMqihK; SameSite=Lax; path=/; expires=Thu, 15-Feb-24 06:45:06 GMT; HttpOnly
Strict-Transport-Security: max-age=31536000; includeSubDomains
Vary: Accept-Encoding
content-security-policy: frame-ancestors 'none'; default-src https: 'unsafe-inline'
x-content-type-options: nosniff
x-frame-options: DENY
x-xss-protection: 1; mode=block
Set-Cookie: __cfruid=207b5db1c54f97fea9eff72b04b07dd999866252-1707970505; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
Server: cloudflare

{
  "result": null,
  "success": false,
  "errors": [
    {
      "code": 12130,
      "message": "access.api.error.invalid_request: allow_authenticate_via_warp cannot be set if the Warp Auth Session Duration is not set for the account"
    }
  ],
  "messages": []
}

```

Reply:
good call out - we don't want to send this if it isn't enabled on the account, the warp_auth_session_duration on the organization is essentially the feature flag

Reply:
I probably didn't notice this when running tests on my account since i have the feature enabled 😅